File: common/src/main/java/org/sqlunet/browser/Application.java
/*
 * Copyright (c) 2020. Bernard Bou <1313ou@gmail.com>.
 */

package org.sqlunet.browser;

import org.sqlunet.settings.Settings;

public class Application extends android.app.Application
{
	@Override
	public void onCreate()
	{
		super.onCreate();
		Settings.initialize(this);
	}
}

File: browserwn/src/main/java/org/sqlunet/browser/wn/Application.kt
/*
 * Copyright (c) 2023. Bernard Bou
 */
package org.sqlunet.browser.wn

import org.sqlunet.browser.wn.common.CommonApplication

class Application : CommonApplication() {

    /**
     * Drop data
     *
     * @return true if flagged in build config
     */
    override fun dropData(): Boolean {
        return BuildConfig.DROP_DATA
    }

    /**
     * Build time
     *
     * @return build time
     */
    override fun buildTime(): String {
        return BuildConfig.BUILD_TIME
    }

    /**
     * Git hash
     *
     * @return git hasj
     */
    override fun gitHash(): String {
        return BuildConfig.GIT_HASH
    }
}
